tests/run-make/c-link-to-rust-dylib/rmake.rs

```diff
@@ -0,0 +1,47 @@
+// This test checks that C linking with Rust does not encounter any errors, with dynamic libraries.
+// See <https://github.com/rust-lang/rust/issues/10434>.
+
+// @ignore-cross-compile
```

Comment:
Problem: unfortunately, this is not a well-formed compiletest directive and does nothing because it's parsed as a normal comment... Directives need to start with `//@` not `// @`.

```suggestion
//@ ignore-cross-compile
```

It's hard to spot and it's not your fault: we should be better at detecting issues like these and report it in compiletest :3

Reply:
I opened https://github.com/rust-lang/rust/issues/125813 so we (mostly I) don't forget to add some help here.

Reply:
Thanks for opening the issue. Fixing it right away!